src/model/ACSPData.ts

```diff
@@ -1,27 +1,23 @@
 import { AMLSupervisoryBodies } from "./AMLSupervisoryBodies";
 import { Address } from "./Address";
 import { Company } from "./Company";
-import { Nationality } from "./Nationality";
 import { RoleType } from "./RoleType";
 import { SectorOfWork } from "./BusinessSector";
 import { TypeOfBusiness } from "./TypeOfBusiness";
+import { ApplicantDetails } from "./ApplicantDetails";
 
 export interface ACSPData {
     id: string;
-    firstName?: string;
-    lastName?: string;
     addresses?: Array<Address>;
     address?: Address;
-    typeOfBusiness? : TypeOfBusiness;
-    roleType? : RoleType;
-    dateOfBirth? : Date;
-    verified? : boolean;
-    nationality? : Array<Nationality>;
-    countryOfResidence? : string;
+    typeOfBusiness?: TypeOfBusiness;
```

Comment:
Are we using this Model? I didn't think so.

Reply:
We are not but I made a change to keep it in sync. But happy to remove it